plugins/GSdx/GSRendererHW.cpp

```diff
@@ -55,7 +55,8 @@ void GSRendererHW::SetScaling()
 	// Framebuffer width is always a multiple of 64 so at certain cases it can't cover some weird width values.
 	// 480P , 576P use width as 720 which is not referencable by FBW * 64. so it produces 704 ( the closest value multiple by 64).
 	// In such cases, let's just use the CRTC width.
-	int fb_width = max((int)m_context->FRAME.FBW * 64, crtc_width);
+	auto search_fb_width = minmax({(int)m_context->FRAME.FBW * 64, crtc_width , 512});
+	int fb_width = search_fb_width.second;
```

Comment:
May I suggest to use std::max instead. I think it could work with ilist. Otherwise just uses 2 max.

As a side note, I think it could be a good idea to cache the fb_* value in the object. And ensure that new size is always bigger than previous size. (Or you can use m_width and m_height).

Something like that
```
int upscaled_fb_h = max ( fb_height * m_upscale_multiplier, m_height).
good_rt_size = (m_height == upscaled_fb_h) && ... 
```